CMakeLists.txt

```diff
@@ -1,76 +1,80 @@
 cmake_minimum_required(VERSION 2.8.12.2 FATAL_ERROR)
-
 project(prometheus-cpp)
+
 enable_testing()
-include(GNUInstallDirs)
 
+include(GNUInstallDirs)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_LIST_DIR}/cmake)
 
 if(NOT DEFINED CMAKE_CXX_STANDARD AND UNIX)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+	set(CMAKE_CXX_STANDARD 11)
```

Comment:
Setting `CMAKE_CXX_STANDARD` won't work with CMake 2.8.12. Please revert

Reply:
Oh, oops. This was by accident when I was fooling around. I will revert it.